recipe/build.sh

```diff
@@ -148,6 +151,9 @@ elif [[ ${cuda_compiler_version} != "None" ]]; then
         export TORCH_CUDA_ARCH_LIST="5.0;6.0;6.1;7.0;7.5;8.0;8.6;8.9;9.0+PTX"
         # $CUDA_HOME not set in CUDA 12.0. Using $PREFIX
         export CUDA_TOOLKIT_ROOT_DIR="${PREFIX}"
+        if [[ "$target_platform" == "linux-aarch64" ]]; then
+	      export CUDA_TOOLKIT_ROOT=${PREFIX}
```

Comment:
Why is this unique to aarch64?

Reply:
It's unique to the cross compiling. To be honest, I don't know the reason, but this environmental variable was written  in the CMake seven years ago, and PyTorch vendors it

https://gitlab.kitware.com/cmake/cmake/-/blame/master/Modules/FindCUDA.cmake#L891

Reply:
Thanks Jinzhe! 🙏

It is worth noting that CMake's [`FindCUDA`]( https://cmake.org/cmake/help/latest/module/FindCUDA.html ) is [deprecated]( https://cmake.org/cmake/help/latest/policy/CMP0146.html )

The path CMake now recommends is to use [`FindCUDAToolkit`]( https://cmake.org/cmake/help/latest/module/FindCUDAToolkit.html ), which also has better cross-compilation support (with a design informed in part by our own use case 😉)

We should encourage PyTorch to migrate

Reply:
Looks like there is an existing issue: https://github.com/pytorch/pytorch/issues/76082